frontend/src/components/eventscard/Card.js

```diff
@@ -0,0 +1,48 @@
+import React, { Component, Fragment } from "react";
+import PropTypes from 'prop-types';
+
+import "../../styles/components/Events.scss";
+import "../../styles/animations/Animation.scss";
+
+
+export default class Card extends Component {
+  static defaultProps = {
+    src: null,
+    title: null,
+  };
+  static propTypes = {
+    src: PropTypes.string,
+    title: PropTypes.string,
+  };
+    render() {
+      const {
+        src, title
+      } = this.props;
+        return (
+            <Fragment>
+                <div
+                  data-wow-duration="1s"
+                  className="event fadeInDown"
+                >
+                  <div className="card1">
+                    <div className="card1-item card1-front">
+                      <img src={src} alt="cultural" />
```

Comment:
Make the value of alt property dynamic. Here it is static having value cultural.